android_webview/test/shell/tango/jni/TangoHandler.cpp

```diff
@@ -746,6 +748,49 @@ bool TangoHandler::getPlanes(std::vector<Plane>& planes) {
   return false;
 }
 
+bool TangoHandler::getPlaneDeltas(PlaneDeltas& planeDeltas) {
+  if (!connected)
+  {
+    return false;
+  }
+
+  std::vector<Plane> planes;
+  getPlanes(planes);
+
+  // Find all the planes that have been added or changed this frame.
+  std::unordered_map<long, Plane> planeMapNew;
+  for (auto plane : planes) {
+    planeMapNew[plane.identifier] = plane;
+    // Check if this is a plane we already know about.
+    auto prev = planeMap.find(plane.identifier);
```

Comment:
Just as a note, C++ STL maps allow to access the content with the [] operator as planeMap[plane.identifier] if you prefer this notation.

Reply:
are you sure it doesn't cause an error if the key doesn't exist? I think using find makes sense to me here - it's more self-documenting.

Reply:
You are right and in this case you need find. For adding/inserting, I recommend using [].